Mothercell/Hive.IO/Hive.IO/DataHandling/Results.cs

```diff
@@ -183,7 +186,7 @@ public void SetSupplySystemsCapacity(string [] supplyTechNames, bool [,] supplyT
             if (supplyTechNames == null || supplyTypes == null || supplyCapacities == null || supplyCapUnits == null)
                 return;
 
-            int shortestLength = int.MaxValue;
+            var shortestLength = int.MaxValue;
```

Comment:
I know, we are making sure this is a `int` by `int.MaxValue`. But since `shortestLength` should always be an `int`, no matter where it is assigned, wouldn't it be clearer to also declare it as `int shortestLength` instead of with `var`?

Reply:
`var` can only be used for local variables if I remember correctly. and it's totally equivalent to `int`, just that it reduces a bit of the "noise" of having types everywhere. The ReSharper plugin (check it out! you can get a free academic license with your ETH account!) prefers `var` and I like to go over the tips the plugin gives - it really helps improve your coding.

No. You can't just assign another type to that variable. `var` is not the same as `dynamic` - the type is specified at _compile time_. This syntax is just much easier to read, especially with longer type names (`var visualizer` vs. `GHVisualizer visualizer`) and basically just removes redundancy.

Reply:
thanks for the explanation. ok I will check out resharper!